### ISSUE TITLE
[19w23d] Symmetry mode turns off after save>exit>load

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Skywanderers#1260]
### Short Description of the issue:
Not sure if this actually is a bug or just how it works, but just in case: Ships do not remember symmetry mode's setting for them when the game is saved, exited, and then loaded again.

### Steps to reproduce the issue:
Enable symmetry mode on ship, save, exit game, load world again. Symmetry mode will be off.

### Environment
- Skywanderers version: 19w23d
- Graphics Card: NVIDIA GeForce GTX 1060 6GB

### Additional information:
[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3284082/output_log.txt)

**Comment by tsunamayo** [comment]
Yes correct, I was planning to add this feature so I keep you posted. Thanks
